### ISSUE TITLE
Cannot get routes

### ISSUE THREAD

**Issue by e-minguez** [kxr/o-must-gather#36]
```
$ omg get routes -n openshift-authentication oauth-openshift -o yaml
usage: omg [-h] {use,project,projects,get,describe,log,logs,whoami,version,machine-config} ...
omg: error: unrecognized arguments: oauth-openshift
```

**Comment by kxr** [comment]
@e-minguez Thank you for pointing this out. This will be fixed in the next release, as we are moving from `argparse` to `click` python library for parsing arguments (thanks to @bostrt). The master branch (which is using click) works fine:

```
$ omg get routes -n openshift-authentication oauth-openshift -o yaml
apiVersion: route.openshift.io/v1
kind: Route
metadata:
  creationTimestamp: '2020-10-29T23:08:09Z'
  labels:
    app: oauth-openshift
  name: oauth-openshift
  namespace: openshift-authentication
  resourceVersion: '14830'
<snip>
```

**Comment by kxr** [comment]
Fixed in release v1.2.0
